mantidimaging/gui/windows/live_viewer/model.py

```diff
@@ -87,14 +91,37 @@ def _handle_file_change(self, file):
                     last_modified_image = max(self.images, key=lambda x: x.stat().st_mtime)
                     self.image_changed.emit(str(last_modified_image))
 
+    def _try_to_read_file(self, file_path) -> bool:
+        """Try and read file, return True if successful and False otherwise"""
+        try:
+            with tifffile.TiffFile(file_path) as tif:
+                tif.asarray()
+                tif.close()
+        except Exception:
+            return False
+        return True
```

Comment:
Would it work to drop this, and just let it fail in the presenter? It would save doing the load twice.

Reply:
should do!